### DISCUSSION TITLE
How to get rid of this weird focus border ?

### DISCUSSION THREAD

**Discussion by jaymehta-twilio** [twilio-labs/paste#1099] [Q&A] [answered]
Hello, I'm using Grid layout with multiple columns as shown in the attached snapshot. With focus or even clicking anywhere in the empty space, this weird looking border appears. Any hints to resolve it would be appreciated. 🙏 

<img width="952" alt="Screen Shot 2021-01-15 at 12 36 32 AM" src="https://user-images.githubusercontent.com/52717786/104701023-e6787000-56c9-11eb-9513-418380957f90.png">

**Comment by TheSisb** [comment, marked as answer]
The focus is expected behavior for accessibility. Related discussion: https://github.com/twilio-labs/paste/discussions/995

If the concern is the shape of the focus, which is understandable, I believe it's because the Grid component uses negative margins. You may be able to work around that by providing some margin of your own to that section. Definitely something for us to consider and we may revise the use of negative margins for this reason.
